test/scssTests.js

```diff
@@ -7,29 +7,29 @@ var fs = require('fs'),
     connect = require('connect'),
     middleware = require('../middleware'),
     fixture = path.join.bind(null, __dirname, 'fixtures'),
-    test_cssFile = fixture('test.css'),
-    test_scssFile = fixture('test.scss'),
-    index_cssFile = fixture('index.css'),
-    index_sassFile = fixture('index.sass'),
-    index_scssFile = fixture('index.scss'),
-    index_sourceMap = fixture('index.css.map'),
+    testCssFile = fixture('test.css'),
+    testScssFile = fixture('test.scss'),
+    indexCssFile = fixture('index.css'),
+    indexSassFile = fixture('index.sass'),
+    indexScssFile = fixture('index.scss'),
+    indexSourceMap = fixture('index.css.map'),
     testUtils = require('./testUtils');
 
 describe('Using middleware to compile .scss', function() {
   var server = connect()
     .use(middleware({
       src: fixture(),
-      dest: fixture()
+      dest: fixture(),
+      indentedSyntax: false
```

Comment:
Added `indentedSyntax: false` in `sassTests.js` (although the default value of said option is `false`), to minimize the diff between `sassTests.js` and `scssTest.js` files.